Remove some source which we don't use, to save a little bit of disk space.

Index: libedataserver/Makefile.am
===================================================================
--- libedataserver/Makefile.am.orig	2008-04-23 17:00:08.000000000 +0100
+++ libedataserver/Makefile.am	2008-04-23 17:01:49.000000000 +0100
@@ -18,18 +18,13 @@
 
 libedataserver_1_2_la_SOURCES =		\
 	$(BONOBO_SOURCES)		\
-	e-account-list.c		\
-	e-account.c			\
-	e-categories.c			\
 	e-data-server-module.c		\
 	e-file-cache.c			\
 	e-flag.c			\
-	e-iconv.c			\
 	e-iterator.c			\
 	e-list.c			\
 	e-list-iterator.c		\
 	e-memory.c			\
-	e-msgport.c			\
 	e-sexp.c			\
 	e-source-group.c		\
 	e-source-list.c			\
@@ -38,7 +33,6 @@
 	e-uid.c				\
 	e-url.c				\
 	e-data-server-util.c		\
-	e-trie.c			\
 	e-xml-utils.c			\
 	e-xml-hash-utils.c		\
 	libedataserver-private.h	\
@@ -56,25 +50,18 @@
 libedataserverincludedir = $(privincludedir)/libedataserver
 
 libedataserverinclude_HEADERS =		\
-	e-account-list.h		\
-	e-account.h			\
-	e-categories.h			\
-	e-component-listener.h		\
 	e-data-server-module.h		\
 	e-file-cache.h			\
 	e-flag.h			\
-	e-iconv.h			\
 	e-iterator.h			\
 	e-list.h			\
 	e-list-iterator.h		\
 	e-memory.h			\
-	e-msgport.h			\
 	e-sexp.h			\
 	e-source-group.h		\
 	e-source-list.h			\
 	e-source.h			\
 	e-time-utils.h			\
-	e-trie.h			\
 	e-uid.h				\
 	e-url.h				\
 	e-data-server-util.h		\
